packages/svelte/src/compiler/phases/3-transform/types.d.ts

```diff
@@ -1,10 +1,12 @@
 import type { Scope } from '../scope.js';
-import type { SvelteNode, ValidatedModuleCompileOptions } from '#compiler';
+import type { Binding, SvelteNode, ValidatedModuleCompileOptions } from '#compiler';
 import type { Analysis } from '../types.js';
+import type { Expression, Identifier } from 'estree';
 
 export interface TransformState {
 	readonly analysis: Analysis;
 	readonly options: ValidatedModuleCompileOptions;
 	readonly scope: Scope;
 	readonly scopes: Map<SvelteNode, Scope>;
+	readonly getters: Record<string, Expression | ((id: Identifier) => Expression)>;
```

Comment:
Would be good to have some JSdoc for this

Reply:
done